compose/arm64.yml

```diff
@@ -0,0 +1 @@
+aarch64.yml
```

Comment:
I can't understand this line.

If its meaning is simply `include aarch64.yml` I would move the content of `aarch64.yml` here (and remove the other file).

Reply:
No, no, it's [a symlink](https://en.wikipedia.org/wiki/Symbolic_link). :) Because:

* `uname -m` returns `arm64` on macOS on Apple Silicon,
* and `aarch64` on AArch64 Linux

It's an unfortunate discrepancy, but both are correct, and mean the same. I though symlink would work well (and it does)